vignettes/opt-dead-code.Rmd

```diff
@@ -30,6 +30,10 @@ speed_up <- function(obj) {
 
 # Dead Code Elimination
 
+## Idea
+
+Dead code are pieces of code that do not affect the output of a program. The removal of these unnecessary code lines can be defined as Dead Code Elimination. These code lines do not serve any purpose in the main program flow because these codes will never be executed mainly because the condition for their execution is logically impossible/infeasible. This removal results in perfomance enhancements. 
```

Comment:
typo 'perfomance' ~> 'performance'.

Reply:
Hi, I just fixed that typo, please have a look...
 Maybe I need a better text editor 😞